packages/core/content-releases/admin/src/components/ReleasesGrid.tsx

```diff
@@ -0,0 +1,87 @@
+import {
+  Box,
+  EmptyStateLayout,
+  Flex,
+  Grid,
+  GridItem,
+  Loader,
+  Typography,
+} from '@strapi/design-system';
+import { AnErrorOccurred } from '@strapi/helper-plugin';
+import { EmptyDocuments } from '@strapi/icons';
+import { useIntl } from 'react-intl';
+import { useHistory } from 'react-router-dom';
+
+import { Release } from '../../../shared/contracts/releases';
+
+interface ReleasesGridProps {
+  status: 'error' | 'success';
+  sectionTitle: 'pending' | 'done';
+  releases?: Array<Release>;
+}
+
+const ReleasesGrid = ({ status, sectionTitle, releases }: ReleasesGridProps) => {
```

Comment:
```suggestion
const ReleasesGrid = ({ status, sectionTitle, releases = [] }: ReleasesGridProps) => {
```

then you don't have to check it's there.